app/src/main/res/values/strings.xml

```diff
@@ -248,6 +248,7 @@
     <string name="google_qr_export_unrelated">Unrelated QR code found. Try restarting the scanner.</string>
     <string name="google_qr_export_scanned">Scanned %d/%d QR codes</string>
     <string name="google_qr_export_unexpected">Expected QR code #%d, but scanned #%d instead</string>
+    <string name="backup_error_bar_message"><b>Vault backup failed recently</b></string>
```

Comment:
Is it possible to move < b> and < /b> out of the string file to avoid user error in the translations?

Reply:
Is that considered good practice? And if so, how are cases where only part of the string is bold handled?

Reply:
I forgot about the ones we already have, let's keep it like this for now. I'm not sure if it's considered good practice.